project1/css/index.css

```diff
@@ -62,3 +62,90 @@ h1, h2, h3, h4, h5 {
     letter-spacing: 1px;
     margin-bottom: 10px;
 }
+
+.container {
+	width: 800px;
+	margin: 0 auto;
+  }
+  
+  header {
+	width: 100%;
+	display: flex;
+	flex-direction: row;
+  }
+  
+  header nav ul {
+	display: flex;
+	list-style: none;
+	justify-content: space-around;
+	
+  }
+  
+  header nav {
+	width: 600px;
+   
+  }
+  
+  header nav ul li a {
+	text-decoration: none;
+	color: grey;
+	align-items: center;
+  }
+  
+  header .logo {
+	display:flex;
+	flex-direction: row;
+	margin-bottom: 20px;
+	
+  }
+  .cta {
+	  display: flex;
```

Comment:
Try keeping consistent indentation throughout your code. I notice more indentation here when compared to, for example, lines 85, 91, 97.